sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_mlme.c

```diff
@@ -294,13 +293,50 @@ static void _mlme_link_check_ans(gnrc_lorawan_t *mac, uint8_t *p)
     mlme_confirm.link_req.margin = p[1];
     mlme_confirm.link_req.num_gateways = p[2];
 
+    DEBUG("gnrc_lorawan_mlme: LinkCheckAns Margin : %u \n",
+          mlme_confirm.link_req.margin);
+    DEBUG("gnrc_lorawan_mlme: LinkCheckAns GwCnt : %u \n",
+          mlme_confirm.link_req.num_gateways);
+
     mlme_confirm.type = MLME_LINK_CHECK;
     mlme_confirm.status = GNRC_LORAWAN_REQ_STATUS_SUCCESS;
     gnrc_lorawan_mlme_confirm(mac, &mlme_confirm);
 
     mac->mlme.pending_mlme_opts &= ~GNRC_LORAWAN_MLME_OPTS_LINK_CHECK_REQ;
 }
 
+static void _mlme_link_adr_req(gnrc_lorawan_t *mac, uint8_t *p)
+{
+    DEBUG("gnrc_lorawan_mlme: LinkADRReq DataRate_TXPower : DR%u TX%u\n",
+           p[1] >> 4, p[1] & 0x0f);
+    DEBUG("gnrc_lorawan_mlme: LinkADRReq ChMask : %u\n", (p[3] << 8 ) | p[2]);
+    DEBUG("gnrc_lorawan_mlme: LinkADRReq Redundancy : %u\n", p[4]);
+
+    mac->last_dr = p[1] >> 4;
+    mac->channel_mask = (p[3] << 8 ) | p[2];
```

Comment:
where's the code that handles multiple LinkADRReq commands, as specified in the standard? (page 25)

Reply:
Updated.